src/Extensions/CheckComposerUpdatesExtension.php

```diff
@@ -38,6 +38,7 @@ public function updatePackageInfo(array &$installedPackageList)
         // Fetch types of packages that are "allowed" - ie. dependencies that we actually care about
         $allowedTypes = (array) Config::inst()->get(UpdatePackageInfoTask::class, 'allowed_types');
         $composerPackagesAndConstraints = $this->owner->getComposerLoader()->getPackages($allowedTypes);
+        $ignoredPackages = (array) Config::inst()->get(self::class, 'ignored_packages');
```

Comment:
I think it'd be good to look for this on the `UpdatePackageInfoTask` like `$allowedTypes` above. If you could change to that, we should also add the private static to this extension with a short doc block explaining what it's for.

Reply:
Done

Reply:
Would you mind adding the static to this class too please?

```php
/**
 * Lists packages that will be ignored by the Composer update checker
 *
 * @config
 * @var string[]
 */
private static $ignored_packages = [];
```

Or something similar

Reply:
OK :)
Thanks for allowing contributors to make changes @Leapfrognz, I can keep this moving along.